xml/cap_depl_eks.xml

```diff
@@ -370,151 +288,347 @@ Custom TCP Rule    TCP          4443            0.0.0.0/0       CAP WSS
 Custom TCP Rule    TCP          443             0.0.0.0/0       CAP HTTPS
 Custom TCP Rule    TCP          20000-20009     0.0.0.0/0       TCP Routing</screen>
  </sect1>
- <sect1 xml:id="sec.cap.eks-external-ips">
-  <title>Find your <literal>kube.external_ips</literal></title>
-
-  <para>
-   In your EC2 VM List, look up the private IP addresses of one of the nodes,
-   and note the address that is used in its private DNS, which looks like
-   <literal>ip-<replaceable>address</replaceable>.us-west-2.compute.internal</literal>.
-   Also note the public IP address, which you will need for the DOMAIN of the
-   cluster in your &productname; configuration file.
-  </para>
- </sect1>
- <sect1 xml:id="sec.cap.eks-deployment">
-  <title>Configuring and Deploying &productname;</title>
-
-  <para>
-   Follow the instructions in <xref linkend="sec.cap.storageclass-prod"/> to
-   deploy &productname;. When you create your
-   <filename>scf-config-values.yaml</filename> file, make the following
-   changes:
+ 
+ <sect1 xml:id="sec.cap.eks.dns-first">
+     <title>DNS Configuration</title>
+     <para>
+         Creation of the EKS load balancer is triggered by a setting in the &cap; 
+         deployment configuration file. This requires first mapping your cluster
+         domain to a worker node. If you are using the Amazon Route53 service 
+         for DNS, your domain and sub-domains are all CNAMEs.
```

Comment:
I believe it should always be all CNAMEs, regardless if you were using Route53.

Reply:
:heavy_check_mark: